simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/StationMessageSender.java

```diff
@@ -267,6 +269,22 @@ public void sendHeartBeatAndSubscribe(HeartbeatRequest heartbeatRequest, Subscri
         sendMessage(new WebSocketClientInboxMessage.OcppMessage(call.toJson()));
     }
 
+    /**
+     * Sends NotifyReport event asynchronously
```

Comment:
Please remove `asynchronously` as its misleading... this method puts a message into a blocking-queue

Reply:
done